.github/workflows/terraform_v2.yml

```diff
@@ -0,0 +1,118 @@
+name: Terraform
+
+# Runs Terraform in a preview mode when a PR is opened and
+# the changes are applied when it's merged into production.
+# https://learn.hashicorp.com/tutorials/terraform/github-actions
+
+on:
+  workflow_call:
+    inputs:
+      working-directory:
+        required: true
+        type: string
+      config:
+        description: "key value pair separated by a semi-colon"
+        type: string
+        required: true
+      aws_remote_role_arn:
+        description: AWS IAM role to access the Terraform remote state
+        type: string
+        required: false
+        default: "arn:aws:iam::424765940475:role/p-bt-github-terraform-state-role"
+      separator:
+        description: "character using to separate the elements of the CI confguration file"
+        type: string
+        required: false
+        default: "_"
```

Comment:
in `terraform.yml` we use `region` as an input; would we want to add this to this new workflow? I see in the example PR running this, its using `us-east-1` which may be expected

Reply:
nevermind; i dug into it a little more and this is most likely fine